resources/styles/components/course-calendar/header.less

```diff
@@ -33,6 +26,25 @@
         border-top-right-radius: 0;
       }
     }
+
+    .-no-periods-text {
+      display: inline-block;
+      margin: 16px 0;
+    }
+
```

Comment:
okay yah, good point.  i dont know how much we've been sticking to that, but yah we should fix if we see it